recipe/bld.bat

```diff
@@ -20,21 +17,17 @@ ninja -j%CPU_COUNT%
 ninja install
 if %ERRORLEVEL% NEQ 0 exit 1
 
+:: strip "lib" prefix from import libraries, in line with previous naming
+move %LIBRARY_LIB%\libblas.lib %LIBRARY_LIB%\blas.lib
+move %LIBRARY_LIB%\libcblas.lib %LIBRARY_LIB%\cblas.lib
+move %LIBRARY_LIB%\liblapack.lib %LIBRARY_LIB%\lapack.lib
+move %LIBRARY_LIB%\liblapacke.lib %LIBRARY_LIB%\lapacke.lib
+move %LIBRARY_LIB%\libtmglib.lib %LIBRARY_LIB%\tmglib.lib
+
 :: testing with shared libraries does not work - skip them.
 :: This is because: to test that the program exits if wrong parameters are given,
 :: the testsuite overrides the symbol xerbla (xerbla logs the error and exits) with
 :: its own version that reports to the test program in case of an error.
 :: This does not work with dylibs on osx and dlls on windows.
 ctest --output-on-failure -E "x*cblat*"
 if %ERRORLEVEL% NEQ 0 exit 1
-
-for %%i in (blas cblas lapack lapacke) do (
-    dumpbin /exports "%LIBRARY_PREFIX%/bin/lib%%i.dll" > exports%%i.txt
-    echo LIBRARY lib%%i.dll > %%i.def
-    echo EXPORTS >> %%i.def
-    for /f "skip=19 tokens=4" %%A in (exports%%i.txt) do echo %%A >> %%i.def
-    lib /def:%%i.def /out:%%i.lib /machine:x64
-    copy %%i.lib "%LIBRARY_PREFIX%/lib/%%i.lib"
-)
```

Comment:
Do we need to keep `libblas.dll.a` for m2-consumers (effectively inverting the above code)?

Reply:
No, they can use the .lib file just fine.